py/circuitpy_mpconfig.h

```diff
@@ -460,7 +475,6 @@ struct _supervisor_allocation_node;
     FLASH_ROOT_POINTERS \
     KEYPAD_ROOT_POINTERS \
     GAMEPAD_ROOT_POINTERS \
-    BOARD_UART_ROOT_POINTER \
```

Comment:
Even if these are allocated outside the heap, they still need to be scanned and marked during garbage collection, because they may contain object pointers that need to be marked in the gc mark phase. I'm not sure why you took this out, then.